docs-source/content/userguide/managing-domains/domain-resource.md

```diff
@@ -174,9 +174,10 @@ Elements related to specifying and overriding WebLogic domain configuration:
 
 * These elements are under `configuration.model`, only apply if the `domainHomeSourceType` is `FromModel`, and are discussed in [Model in Image]({{< relref "/userguide/managing-domains/model-in-image/_index.md" >}}).
 
-  * `configMap`: Name of a ConfigMap containing the WebLogic Deploy Tooling model.
+  * `configMap`: Optional name of a ConfigMap containing WebLogic Deploy Tooling model YAML files or `.properties` files.
```

Comment:
Optional name of a ConfigMap -> Unclear which is optional: the name or the ConfigMap. 
If it's the ConfigMap, then I would suggest writing it as "(Optional) Name of a ConfigMap containing...
Otherwise, if the name is optional, then keep it as is.

Reply:
Fixed.